cmd/main.go

```diff
@@ -108,10 +108,39 @@ func readYAMLFile() error {
 	if err := decoder.Decode(&baseline); err != nil {
 		return fmt.Errorf("error decoding YAML: %v", err)
 	}
+	var entryIDs []string
+	for i, entry := range baseline.Criteria {
+		// if entry in entryIDs
+		if contains(entryIDs, entry.ID) {
```

Comment:
Prefer the native method here (and drop the custom func on line 135)
```suggestion
		if slices.Contains(entryIDs, entry.ID) {
```